main.py

```diff
@@ -1,38 +1,53 @@
 import generate
 import utils
 import searches
+import typer
```

Comment:
Umesto `typer`-a mozes koristiti built-in pyhton biblioteku `argparse`.

Reply:
a moze, al vidi sta sam mu uradio, mozes i skraceno i imas default i help, neverujem da to moze sa args